source/includes/connection-snippets/x509-snippets/scala-connection.scala

```diff
@@ -0,0 +1,33 @@
+// begin x509 connection
+import org.mongodb.scala._
+import scala.collection.JavaConverters._
+import com.mongodb.connection.ClusterSettings
+import com.mongodb.connection.SslSettings
+import org.mongodb.scala.bson.collection.mutable.Document
+import org.mongodb.scala.bson.BsonString
+import com.mongodb.client.model.FindOneAndUpdateOptions
+
+object MongoDBx509 extends App {
+   System.setProperty("javax.net.ssl.trustStore","client.truststore");
+   System.setProperty("javax.net.ssl.trustStorePassword","<your_password>");
+
+   System.setProperty("javax.net.ssl.keyStore", "client.keystore");
+   System.setProperty("javax.net.ssl.keyStorePassword", "<your_password>");
+
+   // null value will force mongoDB server to use DN from cert
```

Comment:
should change DN to Subject for consistency here

Reply:
Fixed